src/components/button/button.yaml

```diff
@@ -0,0 +1,14 @@
+variants:
```

Comment:
Are variants the same as examples?

After research with GOV.UK developers we changed our equivalent from "fixtures" to "examples". (https://github.com/alphagov/govuk_publishing_components/pull/37)

Reply:
Interesting. was fixtures more obvious than variants? 
yes, variants also referrers to examples

Reply:
We didn't try variants. People referred to them as examples, so we used the term they chose.

There may be some subtle differences with how you're using the YAML files. I think it'd be useful to keep YAML structures the same to keep components interoperable.